gops.yaml

```diff
@@ -0,0 +1,28 @@
+package:
+  name: gops
+  version: 0.3.28
+  epoch: 0
+  description: gops is a command to list and diagnose Go processes currently running on your system.
+  copyright:
+    - license: BSD-3-Clause
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - go
```

Comment:
this isn't needed when using `go/install`

Reply:
TIL! Done, thanks!